tests/templates/kuttl/resources/02-assert.yaml.2

```diff
@@ -6,23 +6,68 @@ timeout: 600
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: test-nifi-node-default
+  name: test-nifi-node-resources-from-role
+spec:
+  template:
+    spec:
+      containers:
+        - name: nifi
+          resources:
+            requests:
+              cpu: 500m
+              memory: 2Gi
+            limits:
+              cpu: "1"
+              memory: 2Gi
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+        - name: vector
+{% endif %}
+status:
+  readyReplicas: 1
+  replicas: 1
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: test-nifi-node-resources-from-role-group
+spec:
+  template:
+    spec:
+      containers:
+        - name: nifi
+          resources:
+            requests:
+              cpu: 500m
+              memory: 3Gi
+            limits:
+              cpu: "1"
+              memory: 3Gi
+{% if lookup('env', 'VECTOR_AGGREGATOR') %}
+        - name: vector
+{% endif %}
 status:
   readyReplicas: 1
   replicas: 1
 ---
-apiVersion: v1
-kind: Pod
+apiVersion: apps/v1
+kind: StatefulSet
 metadata:
-  name: test-nifi-node-default-0
+  name: test-nifi-node-resources-from-pod-overrides
 spec:
-  containers:
-    - name: nifi
-      resources:
-        limits:
-          memory: 2Gi
-        requests:
-          memory: 2Gi
+  template:
+    spec:
+      containers:
+        - name: nifi
+          resources:
+            requests:
+              cpu: 600m
```

Comment:
shouldn't this be 700m? and the limit 1200m?

Reply:
Sure thing, was a leftover

Reply:
Lol this had no effect as the file ended in `.2` instead of `.j2` :shrug: